preload.js

```diff
@@ -267,6 +267,10 @@ window.LokiAPI = new LokiServer({
 });
 
 window.mnemonic = require('./libloki/mnemonic');
+const { WorkerInterface } = require('./js/modules/util_worker_interface');
+
+const worker = new WorkerInterface(path.join(app.getAppPath(), 'js', 'util_worker.js'));
```

Comment:
```suggestion
const utilWorkerPath = path.join(app.getAppPath(), 'js', 'util_worker.js');
const worker = new WorkerInterface(utilWorkerPath);
```
Makes it easier to read and looks cleaner